Rb.Forms.Barcode.iOS/Logger/Log.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using Rb.Forms.Barcode.Pcl.Logger;
+
+namespace Rb.Forms.Barcode.iOS.Logger
+{
+    public static class Log
+    {
+        public static void Debug(this ILog log, string message) 
+        {
+            Debug(log, "{0}", message);
+        }
+
+        public static void Debug(this ILog log, string message, params object[] args) 
+        {
+
+            var prefixedMessage = String.Format("Rb.Forms.Barcode [{0}] {1}", log.GetType().Name, message);
```

Comment:
Android automatically puts brackets around the output source, in this case `Rb.Forms.Barcode`. On ios these will be missing, right?